src/languages/janet/index.ts

```diff
@@ -212,6 +212,17 @@ const janetLanguage: Language = {
         "sorted[Ascii]": "sorted",
         "sorted[Int]": "sorted",
       }),
+      clone((node, type) => {
+        if (["boolean", "integer", "text"].includes(type.kind)) {
+          return node;
+        }
+        if (
+          type.kind === "List" &&
+          ["boolean", "integer", "text"].includes(type.member.kind)
+        ) {
+          return func("array/slice", node, int(0), int(-1));
```

Comment:
`@[;a]` is much shorter.

Reply:
Deep copy: `(thaw(freeze a))`

Reply:
Thanks, I was searching for 10 minutes and wasn't able to find anything.